javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var $lightbulb = document.querySelector('.column');
+var $container = document.querySelector('.container');
+
+$lightbulb.addEventListener('click', toggleLightbulb);
+
+var onOff = 2;
+
+function toggleLightbulb(event) {
+  onOff++;
+  if (onOff % 2 === 1) {
```

Comment:
Rather than using a counter, I would use a boolean to keep track of if the light is on or off 🤓